packages/storefront-api-client/src/storefront-api-client.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  createGraphQLClient,
+  CustomFetchAPI,
+  RequestParams as GQLClientRequestParams,
+  getCurrentSupportedAPIVersions,
```

Comment:
Nit: Should this be `getCurrentSupportedApiVersions`?

Missed this in the `graphql-client` review, sorry!